Ntuplizer/python/templates/electronBranches.py

```diff
@@ -34,9 +34,8 @@
         ),
     bools = cms.PSet(
         IsGap = cms.string('isGap'),
-
-        IsCBVIDtight = cms.string('userInt("CBVIDtight")'),
-        IsCBVIDmedium = cms.string('userInt("CBVIDmedium")'),
-        IsCBVIDloose = cms.string('userInt("CBVIDloose")'),
+        IsCBVIDtight = cms.string('userInt("isCBVIDtight")'),
```

Comment:
The value map embedder embeds everything except ints as userFloats, so I think this is a bug. I'm actually a little surprised it ran.

For the future, it's better to check for userData existence in the branches anyway. So something like
```
IsCBVIDtight = cms.string('? hasUserFloat("isCBVIDtight") ? userFloat("isCBVIDtight") : 0.'),
```
is a bit less fragile.